src/components/calcite-scrim/calcite-scrim.scss

```diff
@@ -25,7 +25,13 @@
   overflow-hidden
```

Comment:
Should the `select-none` be removed from scrim if we allow things inside it?

Reply:
That's a good point.
Right now, content in the default slot is still interact-able (in Chrome) and anything "under" the scrim remains non-interact-able.

I can add some tests.

Reply:
@driskull I added a couple tests to check for click. 

And wow...I totally misunderstood your suggestion.
Yes...we should remove `user-select: none`. Will do.